src/components/share-structure/EditShareStructure.vue

```diff
@@ -244,7 +244,7 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
           })
         ) || 'Class name must be unique')
       rules.push(
-        (v: string) => !(v.split(' ').some(r => this.excludedWordsListForClass.includes(r.toLowerCase()))) ||
+        (v: string) => !(v?.split(' ').some(r => this.excludedWordsListForClass.includes(r.toLowerCase()))) ||
```

Comment:
There were two broken unit tests not related to this ticket:
![fixed unit tests](https://github.com/bcgov/bcrs-shared-components/assets/122301442/f2bc9436-b472-436a-b242-e6277340bad2)

This line and line 260 fixes these two tests.

Reply:
Why is this needed, since line 239 should validate that the value exists?

Reply:
I agree. That's why I was confused. The code with the unit tests should work. However, as above, they don't.
![without optional](https://github.com/bcgov/bcrs-shared-components/assets/122301442/8524d85d-fc11-4ad1-8977-5370aa56984a)

Reply:
Looking at the error messages, I'm guessing `s.name` is null... Maybe the data in the unit test is missing names in the shares or series objects?

Reply:
Even weirder: the test failure says the error is on line 243, but your change is on line 247 !?!?

Reply:
I'll be reverting the change that I made. I'll try to find some other way to fix it.